3D Chess/Assets/ParrelSync/Examples/CustomArgumentExample.cs

```diff
@@ -0,0 +1,31 @@
+﻿// This should be editor only
```

Comment:
The Project tab has two main structures: `Assets` and `Packages`:

![image](https://user-images.githubusercontent.com/1942012/128185160-65885518-dd3b-4318-8e48-284e3d98a88a.png)

My understanding so far was, that `Packages` contains everything that was imported via the `Package Manager` and `Assets` contains everything that was imported via the `Asset Store`. Prefabs for UI, etc. go there as well.
`ParallelSync` is an `Asset` as well (even though - to be fair - installed manually) hence they put themselves into the `Asset` folder.

Reply:
Can we install ParrelSync via UPM? That way we keep Assets cleaner and easier to follow.

Reply:
We can indeed. I have removed it though for now since it actually doesn't make sense in the example template. It has no use here. I will add it again via UPM in the local Realm PR.
--> f8c7386